src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -532,8 +532,17 @@ def forward(
 
 
         >>> def sample_frame_indices(clip_len, frame_sample_rate, seg_len):
+        ...     '''
+        ...     Sample frames from the video.
+        ...     Args:
+        ...         clip_len (`int`): Total number of frames to sample
+        ...         frame_sample_rate (`int`): Frame sample rate
+        ...         seg_len (`int`):  Number of frames in a sample segment
+        ...     Returns:
+        ...         result (`List[int]`): List of sampled video frame indices
+        ...     '''
         ...     converted_len = int(clip_len * frame_sample_rate)
-        ...     end_idx = np.random.randint(converted_len, seg_len)
+        ...     end_idx = np.random.randint(converted_len, converted_len + seg_len)
```

Comment:
I don't think this is correct - we can't select a frame longer than the seqence length